A5 calib (Fixed and finalized voltage calibration)

KaeliAutumn opened the pull request:
This is a request to merge the A5_calib branch with the main AraRoot branch. This branch has two main updates:

- The voltage calibration is properly called within kLatestCalib and other calibration methods that include a voltage calibration to be applied. This is the biggest change! If you want to recreate code using the old kLatestCalib, please use the tag kLatestCalib14to20.
- The A5 timing and voltage files have been updated (they are no longer placeholders).

You can find documentation of this problem on the ARA Slack here: https://askaryanradioarray.slack.com/archives/CCLF3DYMV/p1600195927026800

You can also refer to the presentation I gave in September 2020 documenting the issue: http://ara.physics.wisc.edu/docs/0022/002207/001/AraRoot_voltageCal_issue_09212020.pdf

I've also attached the plot that Dave Besson made describing how this change may effect pointing. More work can be done to quantify how this change may have affected analyses.

Please let me know if there are any questions!
Kaeli

[a.pdf](https://github.com/ara-software/AraRoot/files/6144211/a.pdf)

clark2668 commented:
I'll add one more comment, which is a quote from Dave: 

> yeah, I checked a few other reconstruction algorithms - in all cases, Kaeli's calibration gives about 1-2% efficiency increase and a 10-20% tighter reconstructed vertex in both elevation and azimuth.

Here are links to the other DocDB talks where Kaeli described the calibration process. [DocDB 1896](http://ara.physics.wisc.edu/cgi-bin/DocDB/ShowDocument?docid=1896), [DocDB 1977](http://ara.physics.wisc.edu/cgi-bin/DocDB/ShowDocument?docid=1977), [DocDB 2048](http://ara.physics.wisc.edu/cgi-bin/DocDB/ShowDocument?docid=2048), [DocDB 2076](http://ara.physics.wisc.edu/cgi-bin/DocDB/ShowDocument?docid=2076).

Otherwise this looks good, thanks Kaeli!

toej93 commented:
Would it be possible to rename `kLatestCalib14to20 ` to `kLatestCalib14to20Bug ` or something that indicates that there's an issue with using the latter flag? I think being over descriptive is a good thing :)

KaeliAutumn commented:
I updated the variable name as you suggested, Jorge! I also updated the comment next to it in the header file to be extra clear. I'll go ahead and merge now.
Best,
Kaeli
